pkgs/development/libraries/fribidi/default.nix

```diff
@@ -1,21 +1,29 @@
-{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig }:
+{ stdenv
+, fetchFromGitHub
+
+, removeReferencesTo
+, meson
+, ninja
+, pkgconfig
+}:
 
 stdenv.mkDerivation rec {
-  name = "fribidi-${version}";
-  version = "0.19.7";
+  name = "${pName}-${version}";
+  pName = "fribidi";
+  version = "1.0.3";
 
   src = fetchFromGitHub {
-    owner = "fribidi";
-    repo = "fribidi";
-    rev = version;
-    sha256 = "10q5jfch5qzrj2w4fbkr086ank66plx8hp7ra9a01irj80pbk96d";
+    owner = pName;
+    repo = pName;
+    rev = "v${version}";
+    sha256 = "02483nscxc695j9b92clcdf0xb7xkfjry09kqdkkhkzl3vdcj039";
   };
 
-  nativeBuildInputs = [ autoreconfHook pkgconfig ];
-
-  # Configure script checks for glib, but it is only used for tests.
+  nativeBuildInputs = [ removeReferencesTo meson ninja pkgconfig ];
 
-  outputs = [ "out" "devdoc" ];
+  # FIXME: 2016-06-01 v1.0.3: On new release please permit docs generation when
+  # https://github.com/fribidi/fribidi/issues/79 meson can build without c2man
```

Comment:
The pull request has been now merged. Could you add https://github.com/fribidi/fribidi/commit/a465eef7af30217e339eb50947d3256a95cef78e.patch to patches and re-add `devdoc` to outputs?